public/app/features/users/UsersTable.tsx

```diff
@@ -90,11 +90,13 @@ const UsersTable: FC<Props> = (props) => {
                     <UserRolePicker
                       userId={user.userId}
                       orgId={orgId}
-                      builtInRole={user.role}
-                      onBuiltinRoleChange={(newRole) => onRoleChange(newRole, user)}
                       roleOptions={roleOptions}
                       builtInRoles={builtinRoles}
-                      disabled={!contextSrv.hasPermissionInMetadata(AccessControlAction.OrgUsersWrite, user)}
+                      builtInRole={user.role}
+                      onBuiltinRoleChange={(newRole) => onRoleChange(newRole, user)}
+                      builtinRolesDisabled={
+                        !contextSrv.hasPermissionInMetadata(AccessControlAction.OrgUsersWrite, user)
```

Comment:
nit: not for this PR, but I wonder if we should refactor the naming of `builtin roles` to be `basic roles` instead. Otherwise we keep using the two interchangeably, and it's confusing.

Reply:
Yes I started doing that but reverted the changes, easier to do it in another pr